lib/awful/spawn.lua

```diff
@@ -370,6 +370,18 @@ function spawn.easy_async(cmd, callback)
     })
 end
 
+--- Call spawn.easy_async with a shell.
+-- This calls `cmd` with `$SHELL -c` (via `awful.util.shell`).
+-- @tparam string cmd The command.
+-- @tab callback Callback function.
```

Comment:
This is wrong, it's a function. `@tab` is for table. Also, the callback parameters are missing. Please copy/paste the parameters from `easy_async`.